Da geht was schief - Teste zuerst lokal und fixe than für gh action workflow

Run docker-compose up -d
  docker-compose up -d
  # We start Ollama via docker-compose as defined in the project,
  # but the install/llm.sh might also be used for local setup.
  # For CI, we'll rely on docker-compose for the service.
  shell: /usr/bin/bash -e {0}
  env:
    JAVA_HOME: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.18-8/x64
    JAVA_HOME_17_X64: /opt/hostedtoolcache/Java_Temurin-Hotspot_jdk/17.0.18-8/x64
/home/runner/work/_temp/2ef7071e-51a4-4a40-8318-b322414b72ea.sh: line 1: docker-compose: command not found
Error: Process completed with exit code 127.

--- a/.github/workflows/test-integration.yml
+++ b/.github/workflows/test-integration.yml
@@ -37,14 +37,14 @@ jobs:
     - name: Start Services (Database and Ollama)
       run: |
         docker compose up -d
-        # We start Ollama via docker-compose as defined in the project,
+        # We start Ollama via docker compose as defined in the project,
         # but the install/llm.sh might also be used for local setup.
-        # For CI, we'll rely on docker-compose for the service.
+        # For CI, we'll rely on docker compose for the service.
 
     - name: Wait for Oracle Database
       run: |
         echo "Waiting for Oracle Database to be healthy..."
-        # The docker-compose has a healthcheck, we can wait for it.
+        # The docker compose has a healthcheck, we can wait for it.
         MAX_RETRIES=60
         COUNT=0
         until [ "$(docker inspect --format='{{.State.Health.Status}}' oracle-db)" == "healthy" ]; do
@@ -72,7 +72,7 @@ jobs:
         DB_CONN_STR: "system/password@localhost:1521/FREEPDB1"
       run: |
         # Ensure we can connect to the DB from the host
-        # The docker-compose maps 1521:1521
+        # The docker compose maps 1521:1521
         bash test.sh
 
     - name: Upload Test Report to Summary

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ Dieses Repository dient dem Testen des Zusammenspiels eines lokalen LLM mit eine
 ### 1. Umgebung starten
 Um die Datenbank und Ollama zu starten, verwende Docker Compose:
 ```bash
-docker-compose up -d
+docker compose up -d
 ```
 
 ### 2. LLM Modell vorbereiten

